java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -325,12 +311,10 @@ disabled.modules=\
     org.netbeans.modules.xml.catalog.ui,\
     org.netbeans.modules.xml.jaxb,\
     org.netbeans.modules.xml.jaxb.api,\
-    org.netbeans.modules.xml.multiview,\
```

Comment:
A bit too many new dependencies. Could some of them be eliminated by restructuring the modules? I am sure you don't really need `xml.multiview` in VSCode. Why do we need `schema2beans`? `eclipselink`?

I understand that currently the needed modules (`*.persistance`?) have such dependencies. What it would take to eliminate some of them?

Reply:
The problem is that the `j2ee.persistence` module is huge and its split into  `j2ee.persistence.base` and  `j2ee.persistence` would require a substantial amount of time. We should IMHO plan some NBLS module cleanup for the next release.

Reply:
I filed [NETBEANS-6295](https://issues.apache.org/jira/browse/NETBEANS-6295) ... the LSP is getting bloated. We should also invent a way how the project- db- etc supports can plug into the basic LSP protocol server so that we don't increase the dependency nexus. So filed also [NETBEANS-6296](https://issues.apache.org/jira/browse/NETBEANS-6296)